src/docs/product/performance/widgets.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Performance Widgets"
+sidebar_order: 100
+description: "Learn more about Sentry's Performance widgets such as, Most Related Issues, Most Regressed Transactions, Slow HTTP Ops, and how they can be used to surface performance insights in your applications."
+---
+
+[Performance](/product/performance/) widgets offer visualizations that you can change to best match your workflow. They provide performance-specific views that allow you see and act on transaction data across your organization.
+Depending on which **Performance** tab you've selected, different curated performance widgets will be available.
+
+## Most Improved
+
+Shows the most improved transactions and visualizes the improvement for the selected transaction in the list. Only trends of a specific significance and throughput are shown. Learn more in [Trends](/product/performance/trends).
+
+## Most Regressed
+
+Shows the most regressed transactions and visualizes the regression for the selected transaction in the list. Only trends of a specific significance and throughput are shown. Learn more in [Trends](/product/performance/trends).
+
+## Most Related Issues
+
+Shows the highest count of any issue group related to a transaction and visualizes the issue count over time.
+
+## Most Slow Frames
+
+Shows transactions with the most slow frames for mobile projects. Learn more in [Mobile Vitals](/product/performance/mobile-vitals).
+
+## Most Frozen Frames
+
+Shows transactions with the most frozen frames for mobile projects. Learn more in [Mobile Vitals](/product/performance/mobile-vitals).
+
+## Worst LCP Vitals
+
+Shows transactions with the highest count of poor LCP (Largest Contentful Paint) and visualizes the proportions of user experience for this vital. Learn more in [Web Vitals](/product/performance/web-vitals).
+
+## Worst FCP Vitals
+
+Shows transactions with the highest count of poor FCP (First Contentful Paint) and visualizes the split in vital performance to illustrate the proportions of user experience. Learn more in [Web Vitals](/product/performance/web-vitals).
+
+## Worst FID Vitals
+
+Shows transactions with the highest count of poor FID (First Input Delay) and visualizes the split in vital performance to illustrate the proportions of user experience. Learn more in [Web Vitals](/product/performance/web-vitals).
+
+## Slow HTTP Ops
+
+Shows transactions with slow cumulative HTTP spans. Slow HTTP Ops are note shown for all projects and transactions as the transactions must contain HTTP spans. Check out the SDK documentation for the project in question for more information.
+
+## Slow DB Ops
+
+Shows transactions with slow cumulative DB spans. Slow DB Ops are not shown for all projects and transactions as the transactions must contain DB spans. Check out the SDK documentation for the project in question for more information.
```

Comment:
Just changing the capitalization since we're talking about a general concept here and not a UI element.

```suggestion
## Slow HTTP Ops

Shows transactions with slow cumulative HTTP spans. Slow HTTP ops are note shown for all projects and transactions as the transactions must contain HTTP spans. Check out the SDK documentation for the project in question for more information.

## Slow DB Ops

Shows transactions with slow cumulative DB spans. Slow DB ops are not shown for all projects and transactions as the transactions must contain DB spans. Check out the SDK documentation for the project in question for more information.
```